Update UserGuide

qinxutan opened the pull request.

qinxutan committed:
Merge pull request #14 from AY2324S2-CS2103T-T09-4/master

Save classes to database and Update UI (#65)

qinxutan committed:
Merge pull request #15 from AY2324S2-CS2103T-T09-4/master

Merge

qinxutan committed:
Merge pull request #17 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Merge pull request #18 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Merge pull request #19 from AY2324S2-CS2103T-T09-4/master

Add command to sort students (#117)

qinxutan committed:
Merge pull request #20 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Merge pull request #21 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Merge pull request #22 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Merge pull request #23 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Update UG

whelan-low commented on docs/UserGuide.md:
```diff
@@ -243,12 +283,21 @@ Examples:
+| **Add new team**                        | `add_team module/<moduleCode> tutorial/<tutorialClass> name/<team_name> size/<team_size>` <br> e.g., `add_team module/CS2103T tutorial/T09 name/Team 1 size/5`                                                                         |
+| **Delete team**                         | `delete_team module/<moduleCode> tutorial/<tutorialClass> name/<team_name>` <br> e.g., `delete_team module/CS2103 tutorial/T09 name/Team 4`                                                                                            |
+| **View team**                           | `view_teams <index/, name/> <index or name> module/<moduleCode> tutorial/<tutorialClass>` <br> e.g., `view_teams name/Team 1 module/CS2103T tutorial/T09`                                                                              |
+| **Randomly allocate team**              | `/random_teams module/<module> tutorial/<tutorialClass> teams/<number_of_teams>` <br> e.g., `random_teams module/CS2103 tutorial/T09 teams/4`                                                                                          |
```
Theres also allocate student to team and delete student from team commands missing

qinxutan committed:
Delete student from team (#137) (#24)

* Add assertion test

* Add delete student from team command

* Add more test cases and fix /delete_student_from_team message formatting

* Add successful test cases

* Fix error message for delete by id command

* Fix merge issues

* Put delete student from team success message back into DeleteStudentFromTeamCommand class

* Move getModuleAndTutorialPair into ModuleTutorialPair class, fixed magic number issues and implementation errors

Co-authored-by: Whelan <111336032+whelan-low@users.noreply.github.com>

qinxutan committed:
Update (#25)

* Delete student from team (#137)

* Add assertion test

* Add delete student from team command

* Add more test cases and fix /delete_student_from_team message formatting

* Add successful test cases

* Fix error message for delete by id command

* Fix merge issues

* Put delete student from team success message back into DeleteStudentFromTeamCommand class

* Move getModuleAndTutorialPair into ModuleTutorialPair class, fixed magic number issues and implementation errors

* Update UML  (#130)

* Update DG

* DG Update

* Update UML

* Checkstyle

* Update uml diagram based on pr cmts

---------

Co-authored-by: Jajabonks <84561814+Jajared@users.noreply.github.com>

* Update UserGuide.md (#138)

* Update UserGuide.md

* Update Quick Start section of UserGuide.md

* Update Glossary of UserGuide.md

* Glossary has not been fully updated and more terms can be added to
fully encompasses the unfamiliar terms used in this User Guide.

* Update Add and Allocate student and Random team command in UserGuide.md

* Update add_student, allocate_team and random_teams commands of the
feature section in the user guide.

* Update Purpose and Navigation of User Guide

---------

Co-authored-by: Jajabonks <84561814+Jajared@users.noreply.github.com>

---------

Co-authored-by: Whelan <111336032+whelan-low@users.noreply.github.com>
Co-authored-by: Jajabonks <84561814+Jajared@users.noreply.github.com>
Co-authored-by: Zack-Tay <101039486+Zack-Tay@users.noreply.github.com>

qinxutan committed:
Merge pull request #26 from AY2324S2-CS2103T-T09-4/master

Update

qinxutan committed:
Merge branch 'master' into Update-UG

qinxutan committed:
Merge remote-tracking branch 'origin/master' into Update-UG

qinxutan committed:
Update UG

Zack-Tay reviewed (approved):
LGTM!
